content/Contributing/UITranslations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "String Translations"
+description: "How to contribute CORE text string translations."
+weight: 5
+tags:
+- coretranslate
+- scaletranslate
+---
+
+## How to Contribute CORE and SCALE Text String Translations
+
+{{< hint info >}}
+Users can contribute text string translations for the TrueNAS translation project.  
+The first step is to sign into GitHub. Navigate to the TrueNAS webui directory, the main page of the repository.
+{{< /hint >}}
+
+### Navigate to the Directory
+
+Navigate to this location: [TrueNAS translation directory](https://github.com/truenas/webui/tree/master/src/assets/i18n).  
+The .json language conversion files are located within the "webui/src/assets/i18n" directory.
```

Comment:
Quotes are not preferred due to their getting accidentally pulled into highlight/copy/paste operations so frequently. Bold is a good way to emphasize something specific in a way that allows easy copy/paste.
```suggestion
The .json language conversion files are located within the **webui/src/assets/i18n** directory.
```